frontend/src/components/ReadingSection.tsx

```diff
@@ -412,6 +412,228 @@ export default function ReadingSection() {
     );
   };
 
+  const renderViewpointsArticle = () => {
+    if (!currentPassage || activeSection !== "viewpoints") return null;
+
+    return (
+      <div className="card">
+        <h2 className="text-xl font-bold text-gray-900 mb-4 border-b border-gray-200 pb-2">
+          {currentPassage.title}
+        </h2>
+        <div className="prose max-w-none text-gray-700 leading-relaxed overflow-y-auto max-h-[calc(100vh-12rem)] pr-2">
+          {currentPassage.content
+            .split("\n")
+            .map((paragraph, index) => (
+              <p key={index} className="mb-4">
+                {paragraph}
+              </p>
+            ))}
+        </div>
+        
+        {/* Main Article Questions */}
+        <div className="mt-6 pt-6 border-t border-gray-200">
+          <h3 className="text-lg font-semibold text-gray-900 mb-4">
+            Questions about the Article
+          </h3>
+          <div className="space-y-4">
+            {currentPassage.questions.map((question, index) => (
+              <div key={question.id} className="space-y-2">
+                <div className="flex items-start space-x-3">
+                  <span className="font-medium text-gray-900 mt-1">
+                    {index + 1}.
+                  </span>
+                  <div className="flex-1">
+                    <h4 className="text-sm font-medium text-gray-900 mb-2">
+                      {question.question}
+                    </h4>
+                    <select
+                      value={selectedAnswers[question.id] ?? ""}
+                      onChange={(e) =>
+                        handleAnswerSelect(question.id, parseInt(e.target.value))
+                      }
+                      disabled={showResults}
+                      className={`w-full px-3 py-2 border rounded-md text-sm focus:outline-none focus:ring-2 focus:ring-primary-500 focus:border-primary-500 ${
+                        showResults
+                          ? selectedAnswers[question.id] === question.correctAnswer
+                            ? "bg-green-50 border-green-300 text-green-800"
+                            : selectedAnswers[question.id] !== undefined
+                            ? "bg-red-50 border-red-300 text-red-800"
+                            : "bg-gray-50 border-gray-300"
+                          : selectedAnswers[question.id] !== undefined
+                          ? "bg-primary-50 border-primary-300"
+                          : "bg-white border-gray-300"
+                      }`}
+                    >
+                      <option value="">Choose an option</option>
+                      {question.options.map((option, optionIndex) => (
+                        <option key={optionIndex} value={optionIndex}>
+                          {option}
+                        </option>
+                      ))}
+                    </select>
```

Comment:
This questions section should be part of right section. 

Right section: 
- At top question with header text
- Response section with dropdown after the questions. 
Check attachment for section UI and how to display Right and Left sections. 
<img width="1327" alt="Screenshot 2025-07-01 at 3 10 55 PM" src="https://github.com/user-attachments/assets/8f305403-f68d-4cc3-83a1-0930eaf14614" />